AFNetworking/AFURLRequestSerialization.m

```diff
@@ -578,8 +578,15 @@ - (void)encodeWithCoder:(NSCoder *)coder {
 
 - (instancetype)copyWithZone:(NSZone *)zone {
     AFHTTPRequestSerializer *serializer = [[[self class] allocWithZone:zone] init];
-    dispatch_sync(self.requestHeaderModificationQueue, ^{
-        serializer.mutableHTTPRequestHeaders = [self.mutableHTTPRequestHeaders mutableCopyWithZone:zone];
+
+    // \c self.HTTPRequestHeaders getter is called and not \c self.mutableHTTPRequestHeaders in
+    // order to synchronized the access to the mutable headers dictionary and to ensure we get
+    // the most updated headers.
```

Comment:
I think there's no need to break line here.

Reply:
Done.